### ISSUE TITLE
Make Quiz

### ISSUE THREAD

**Issue by Sebastian2908-2007** [Sebastian2908-2007/villainy-check#58]
* as a user under a paid admin account I should be able to take a quiz. Once the quiz is done I should be able to have my results emailed to my admin.
* so that they can make informed decisions when it comes to their interactions and dealings with me.

***NOTE***
* As of now our quiz page url looks like this /quiz/6529d46fc0f6782d99937a05/user/652c8291d48d2185f0a18da1
* We have the id of the quiz so that we can pull in our quiz data for testing.
* We also have our subject users _id so we can update there data e.g. "quizOutcome" property on the user. This will be the score of the user.
* We will also need to store the admin users _id somehow so that we can email the results of the quiz to them. I was thinking we could store the admins data on the user.

**Comment by Sebastian2908-2007** [comment]
I have added styled UI and Q & functionality. I am handling results distribution and email in a separate issue.
